fendermint/testing/scripts/common.env

```diff
@@ -1,8 +1,10 @@
 CMT_DOCKER_IMAGE="cometbft/cometbft:v0.37.x"
 FM_DOCKER_IMAGE="fendermint:latest"
+PROMTAIL_DOCKER_IMAGE="grafana/promtail:latest"
 CMT_CONTAINER_NAME="${NETWORK_NAME}-cometbft"
 FM_CONTAINER_NAME="${NETWORK_NAME}-fendermint"
 ETHAPI_CONTAINER_NAME="${NETWORK_NAME}-ethapi"
+PROMTAIL_CONTAINER_NAME="${NETWORK_NAME}-promtail"
```

Comment:
So there will be a `smoke-promtail` and a `snapshot-promtail`, which are the defaults, but in the `snapshot-test` it looks like all the further containers which have their individual `node-1.env`, `node-2.env` will not override this name and will try to run a container with the same name. 

I'm sorry, I know this setup is confusing :disappointed:

Reply:
Oh good catch! I think here I could use a single promtail container, and just make sure that each fendermint container generates a unique log file so they won't overwrite each other.